scripts/vmbuild.rb

```diff
@@ -110,6 +110,7 @@ def verify_run(output)
   targets.sort.reverse.each do |target|
     imgfac_target = target.imagefactory_type
     vhd_image     = target.file_extension == "vhd"
+    gce_image     = target.file_extension == "tar.gz"
```

Comment:
It might be better to check for `target` rather than file extension, in case we add tar.gz format for other image types in the future, as renaming image to disk.raw is GCE specific.